site/en/guides/pyspinel/install-wireshark.md

```diff
@@ -0,0 +1,57 @@
+# Install Wireshark
+
+## Linux
+
+Open a terminal and run the following commands to download and install Wireshark:
+
+```
+$ sudo add-apt-repository ppa:wireshark-dev/stable
+$ sudo apt-get update
+$ sudo apt-get install wireshark
+```
+
+We recommend running Wireshark as a non-root user. To do so, reconfigure the package:
```

Comment:
Let put root in code font since we make the distinction for the `wireshark` user

non-`root`